src/server/routes/api/document/indra.js

```diff
@@ -36,18 +69,45 @@ const getDocuments = pairs => {
 
   const transform = o => {
     let pmids = Object.keys( o );
-    let arr = pmids.map( pmid => {
-      let elements = o[ pmid ];
-      // prevent duplication of the same interactions in a document
-      elements = _.uniqWith( elements, _.isEqual );
-      elements.forEach( e => delete e.pmid );
-      return { elements, pmid };
-    } );
-
-    return arr;
+    return tryPromise( () => fetchPubmed({ uids: pmids }) )
```

Comment:
This should check to see if `pmids` is empty.
PubMed EUTILS and the wrapper `fetchPubmed` service will return HTTP 400 status when you call it without including a valid pmid. So when INDRA returns 0 statements, your `getDocuments` ends up throwing the HTTP 400.